.github/PULL_REQUEST_TEMPLATE/projectTemplate.md

```diff
@@ -0,0 +1,8 @@
+## Template Info
+- Name: <!-- replace this comment with the name -->
+- Link to the NuGet Package: <!-- replace this comment with the link to NuGet package-->
+- Underlying SPA framework: <!-- Examples are React, Vue, Angular, ... -->
+
+## Description
+
+<!-- Tell us more about this project template -->
```

Comment:
```suggestion
Please use this template for each submission

1. **Template name** / [GitHub](https://github.com/**owner/repo**) /  [NuGet](https://www.nuget.org/packages/**packageName**/) - Short description of the template
```

The suggested format should be a code block, but I can't do that in a PR suggestion 😆